soc/alif/ensemble/Kconfig.soc

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2025 Alif Semiconductor
+#
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_FAMILY
+	default "ensemble" if SOC_FAMILY_ENSEMBLE
```

Comment:
Again, your're already inside the Ensemble directory so checking for Ensemble is wrong.